apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/internal/ApolloGenerateSourcesTask.kt

```diff
@@ -18,6 +19,13 @@ abstract class ApolloGenerateSourcesTask : DefaultTask() {
   @get:Optional
   abstract val customTypeMapping: MapProperty<String, String>
 
+  @get:Internal
+  abstract val customIdGenerator: Property<CustomIdGenerator>
+
+  @Input
+  @Optional
+  fun customIdGeneratorVersion() = customIdGenerator.orNull?.version
```

Comment:
how should this be used? The UP_TO_DATE checks doesn't say it's updated when version is changed.